packages/editor/src/client/app.tsx

```diff
@@ -129,20 +128,15 @@ export function App() {
   const {current} = useRoute()
 
   const {i18n} = useTranslation()
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  const [lng, setLang] = useState<object>(enGB)
+  const [lng, setLang] = useState<Record<string, any>>(enGB)
 
   i18n.on('languageChanged', lng => {
-    const currentLanguage: any = new Map()
-
-    if (lng === 'fr') {
-      currentLanguage.set(lng, fr)
-    } else if (lng === 'de') {
-      currentLanguage.set(lng, de)
-    } else if (lng === 'en') {
-      currentLanguage.set(lng, en)
-    }
-    setLang(currentLanguage.get(lng))
+    const currentLanguageMap = new Map<string, any>([
```

Comment:
You should initialize `const currentLanguageMap` outside the `languageChanged` function. Otherwise you always reset the map on language change.

Reply:
👍